.coderabbit.yaml

```
@@ -0,0 +1,17 @@
+  profile: 'chill'
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: false
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: false
+  branches:
+    include:
+      - "**"  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)